README.md

```diff
@@ -12,6 +12,16 @@
 
 A community repository for Helm Charts of OpenSearch Project.
 
+```shell
```

Comment:
Sorry to bring it now, but can we not add a title for installation here,

Reply:
Done.